docs/platform_management_plan/project_management.rst

```diff
@@ -77,14 +77,14 @@ Technical committees
   is to do a breakdown of platform sagas to the concrete tasks (see `Planning`_) .
   Currently following *Communities* are defined in the *SCORE* project:
 
-  * *Tooling/Infrastructure*: *community* for all kind of infra topics:
-    compiler, IDE, build toolchain and etc.
+  * *Operational*: *community* for all kind of infra topics:
+    compiler, IDE, build toolchain and etc. See `GitHub Discussions/Operational Community  <https://github.com/orgs/eclipse-score/discussions/categories/operational-community>`_ for more.
   * *Testing*: *community* to clarify questions and define testing strategy
-    for the 'SCORE' project
+    for the 'SCORE' project. See `GitHub Discussions/Testing Community <https://github.com/orgs/eclipse-score/discussions/categories/testing-community>`_ for more.
   * *Software Architecture*: *community* for clarification of software architecture topics,
-    e.g. discussion of new features or coding guidelines.
+    e.g. discussion of new features or coding guidelines. See `GitHub Discussions/Architecture <https://github.com/orgs/eclipse-score/discussions/categories/architecture-community>`_ for more.
   * *Software Development Process*: *community* for definition and maintaining
-    of safety, security and quality software development process.
+    of safety, security and quality software development process. See `GitHub Discussions/SW Dev Process <https://github.com/orgs/eclipse-score/discussions/categories/sw-dev-process-community>`_ for more.
```

Comment:
The operational community and testing community both have the suffix ` Community`, the other two not. We should be consistent and either add it to all entries or remove it.

Reply:
Thanks, true! Fixed.